src/foam/u2/search/IntegerSearchView.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.search',
+  name: 'IntegerSearchView',
+  extends: 'foam.u2.View',
```

Comment:
If you extend Controller instead of View then you will get the exports: 'as data' for free. Generally, a View is like an MVC View which is bound to some data, but in this case the GUI is its own data, which makes it more of a Controller.